### ISSUE TITLE
Multiple values for CLI

### ISSUE THREAD

**Issue by TildeWill** [3YOURMIND/django-migration-linter#229]
I'm trying to execute this:
```
python manage.py lintmigrations --quiet=[ok,ignore] --exclude-apps=[two_factor,reversion]
```
My goal is to only get errors and warnings, and exclude the two apps called two_factor and reversion. But it seems like I can only supply one value. Can someone help me understand how to supply more than one value?

**Comment by TildeWill** [comment]
I was able to work around this by using settings.py, and not having this information in the CLI arguments but I'm still curious as to how this was supposed to work.

**Comment by skarzi** [comment]
hi @TildeWill

have you tried passing multiple arguments by repeating the usage of the flags? Something like this: 

```bash
python manage.py lintmigrations --quiet ok --quiet ignore --exclude-apps two_factor --exclude-apps reversion
```

Probably, also calling this command without these square brackets will work ;)

**Comment by David-Wobrock** [comment]
Hi @TildeWill 

Thank you very much for submitting an issue.

@skarzi repeating the flags will not work I believe. The last one will probably override the first one.

As the documentation/help text is stating:
`--exclude-apps EXCLUDE_APPS [EXCLUDE_APPS ...]`
and
`[-q {ok,ignore,warning,error} [{ok,ignore,warning,error} ...]]`
Therefore it should be
```
python manage.py lintmigrations --quiet ok ignore --exclude-apps two_factor reversion
```
